pkg/plugin/plugin.go

```diff
@@ -754,6 +756,35 @@ func (e *AutoscaleEnforcer) Score(
 	return score, nil
 }
 
+// NormalizeScore weights scores uniformly in the range [minScore, trueScore], where
+// minScore is framework.MinNodeScore + 1.
+func (e *AutoscaleEnforcer) NormalizeScore(
+	ctx context.Context,
+	state *framework.CycleState,
+	pod *corev1.Pod,
+	scores framework.NodeScoreList,
+) *framework.Status {
```

Comment:
It'd also be good to have metrics & logs here about what's going on (i.e. metrics for "NormalizeScore was called!" and logs for "changed score ; node=..., pod=..., old score=..., new score=...").

Reply:
Fixed in 72b53c3a0193e812ea9d19aee2d39ce750ae54ed